engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/JobPreparationActor.scala

```diff
@@ -60,43 +40,24 @@ case class JobPreparationActor(executionData: WorkflowExecutionActorData,
   }
 
   def prepareJobExecutionActor(inputs: Map[LocallyQualifiedName, WdlValue]): Try[BackendJobPreparationSucceeded] = {
+
+    import RuntimeAttributeDefinition.{addDefaultsToAttributes, evaluateRuntimeAttributes}
+    val curriedAddDefaultsToAttributes = addDefaultsToAttributes(factory.runtimeAttributeDefinitions(initializationData), workflowDescriptor.backendDescriptor.workflowOptions) _
```

Comment:
what does the '_' at the end do? Is that what MC is talking about in the comments below?